src/lib/services/client-metrics/ttl-list.test.js

```diff
@@ -43,18 +43,45 @@ test('should slice off list', () => {
         expired.push(entry);
     });
 
+    expect(expired).toHaveLength(0);
+    expect(list.list.toArray()).toHaveLength(4);
+
     jest.advanceTimersByTime(21);
     expect(expired).toHaveLength(1);
+    expect(list.list.toArray()).toHaveLength(3);
 
     jest.advanceTimersByTime(51);
     expect(expired).toHaveLength(2);
+    expect(list.list.toArray()).toHaveLength(2);
 
     jest.advanceTimersByTime(201);
     expect(expired).toHaveLength(3);
+    expect(list.list.toArray()).toHaveLength(1);
 
     jest.advanceTimersByTime(301);
     expect(expired).toHaveLength(4);
+    expect(list.list.toArray()).toHaveLength(0);
 
     list.destroy();
     jest.useRealTimers();
 });
+
+test('should add item created in the past but expiring in the future', () => {
```

Comment:
I added this to ensure it both works with milliseconds and to catch a bug that I almost introduced and the other tests didn't catch